src/lib/utils/showDecorationActionSheet.tsx

```diff
@@ -28,6 +29,14 @@ export default (decoration: Decoration) =>
 			onClose: () => hideActionSheet()
 		},
 		options: [
+			{
+				icon: getAssetIDByName('ic_message_copy'),
+				label: 'Copy Decoration Hash',
+				onPress: () => {
+					clipboard.setString(decoration.hash);
+					showToast('Copied!', getAssetIDByName('toast_copy_message'));
```

Comment:
what do i put here instead of the **Copied!** message

Reply:
"Copied Decoration Hash"